packages/features/form-builder/FormBuilder.tsx

```diff
@@ -531,10 +531,13 @@ function FieldEditDialog({
                       }
                       label={t("identifier")}
                     />
-                    <CheckboxField
-                      description={t("disable_input_if_prefilled")}
-                      {...fieldForm.register("disableOnPrefill", { setValueAs: Boolean })}
-                    />
+
+                    {fieldForm.getValues("name") !== "cancelReason" && (
+                      <CheckboxField
```

Comment:
Why shouldn't the prefill input checkbox be visible for the cancellation reason?